README.md

```diff
@@ -14,9 +14,9 @@ Number             | Title                                                    |
 -------------------|----------------------------------------------------------|-------------------|----------------|--------
 [1](bsip-0001.md)  | BSIP Purpose and Guidelines                              | Fabian Schuh      | Informational  | Draft
 [2](bsip-0002.md)  | Refund Create Order Fees on Cancel                       | Daniel Larimer    | Protocol       | Superseded by [26](bsip-0026.md)
-[3](bsip-0003.md)  | Maker / Taker Market Fees Flag                           | Daniel Larimer    | Protocol       | Deferred
+[3](bsip-0003.md)  | Maker / Taker Market Fees Flag                           | Daniel Larimer    | Protocol       | Superseded by [81](https://github.com/bitshares/bsips/issues/229)
```

Comment:
Hm, do we make a difference between "Superseded" and "Obsoleted"? Otherwise please make wording consistent throughout the document.

Reply:
If something is superseded by something newer, it is replaced because it has become old-fashioned or unacceptable. https://www.collinsdictionary.com/dictionary/english/supersede

I'm an English noob. Does "A superseded by B" implies that B was ever "in use"? Can we just use the word "replaced" which looks simpler?

I feel I'm wasting time discussing this.